frontends/web/src/utils/route.tsx

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
a component for making accesible `route` (marked as deprecated because i don't want people using it, being a transitional solution)